odoo/addons/base/views/res_partner_views.xml

```diff
@@ -267,6 +267,7 @@
                         <page string="Contacts &amp; Addresses" autofocus="autofocus">
                             <field name="child_ids" mode="kanban" context="{'default_parent_id': active_id, 'default_street': street, 'default_street2': street2, 'default_city': city, 'default_state_id': state_id, 'default_zip': zip, 'default_country_id': country_id, 'default_supplier': supplier, 'default_customer': customer, 'default_lang': lang, 'default_user_id': user_id}">
                                 <kanban>
+                                    <field name="id"/>
```

Comment:
Do you actually need to explicitly include the `id` field to use it? I thought it was always available.